src/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const Footer = () => {
+
+    const contactInfo = [
+    "123 Fake Street, London, E14UD",
+    "hello@fakehotel.com",
+    "0123 456789",
+    ];// contact info 
+        return (
+        <div className="the-contact">
+        <ul>
+       {contactInfo.map((item) =>
```

Comment:
it seems that you are missing a key when mapping over the `contactInfo` array in the Footer component. Each element in a React list should have a unique `key` assigned to it to help with rendering. Take a look into [this](https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js)